keyboards/sofle/keyhive/readme.md

```diff
@@ -0,0 +1,66 @@
+# Sofle Keyboard
+
+![SofleKeyboard version 2.1 RGB Keyhive](https://i.imgur.com/WH9OoWuh.jpg)
+
+Sofle is 6×4+5 keys column-staggered split keyboard. Based on Lily58, Corne and Helix keyboards.
+
+For detailed instructions on using VIA and flashing, please refer to the sections further down the page!
+
+For details about the keyboard design, refer to Josef's blog: [Sofle Keyboard - a split keyboard based on Lily58 and Crkbd](https://josef-adamcik.cz/electronics/let-me-introduce-you-sofle-keyboard-split-keyboard-based-on-lily58.html)
+
+Build guide and log is available here: [SofleKeyboard build log/guide](https://josef-adamcik.cz/electronics/soflekeyboard-build-log-and-build-guide.html)
```

Comment:
```suggestion
Build guide: [Keyhive Sofle RGB build guide](https://github.com/keyhive/build_guides/blob/master/docs/keyboards/sofle-rgb.md)
```

The original build guide is not suitable for the Keyhive version of Sofle RGB, and in fact reading https://josefadamcik.github.io/SofleKeyboard/build_guide.html confuses some people so that they solder the controller in a wrong orientation (on the original Sofle V2 boards the controller needs to be soldered with its component side up on the right half, but on the Keyhive Sofle RGB version the controller is soldered with its component side down on both halves).